### DISCUSSION TITLE
Swip off

### DISCUSSION THREAD

**Discussion by Showcase-Joz** [dailydotdev/daily#1023] [Feature Requests]
I want to be able to swipe off or swipe across to remove an item / card from the top of my stack having either read it or choosing to ignore it

**Comment by idoshamun** [comment]
Hi @Showcase-Joz,

Just to be clear you mean on the mobile PWA right?

**Comment by Showcase-Joz** [comment]
I can see nothing to distinguish the type of app. But yes, mobile app,
android. If that makes tit a pwa, super.

Though as a user I'd expect the same/similar action across all forms of the
service 👍

**Reply by idoshamun** [reply]
Sorry about the confusion, yeah I meant mobile in general. But I understand that this is what you use. Thanks for the info :)
